Starting from version 2006.12.08, Gentoo installs live555 as a system
library properly. For policy we also build a shared version of it that
uses PIC.
For this reason, we can reasonably force VLC to create a Plugin for
live555 rather than a builtin.Index: vlc-0.8.6/configure.ac
===================================================================
--- vlc-0.8.6.orig/configure.ac
+++ vlc-0.8.6/configure.ac
@@ -1558,7 +1558,7 @@ if test "${enable_live555}" = "yes"; the
       ], [
         AC_CHECK_LIB(liveMedia, main, [
           # We only have -lliveMedia, do builtins
-          VLC_ADD_BUILTINS([live555])
+          VLC_ADD_PLUGINS([live555])
           VLC_ADD_LDFLAGS([live555], [-lliveMedia -lgroupsock -lBasicUsageEnvironment -lUsageEnvironment])
         ])
       ])
